tensorflow/contrib/cmake/tools/create_deffile.py

```diff
@@ -0,0 +1,107 @@
+"""
+create-datafiles.py - tool to create a windows def file to export
+  symbols from tensorflow.dll to enable tf.load_library().
+  Because the linker allows only 64K sumbols to be exported per dll
+  we filter the symbols down to the essentials. The regular expressions
+  we use for this are specific to tensorflow.
+
+  TODO: this seems to work fine but there is an issue with exporting
+  'const char * const' and importing it from a user_ops. The problem is
+   on the importing end and using __declspec(dllimport) works around it.
+"""
+import argparse
+import io
+import os
+import re
+import sys
+import tempfile
+from subprocess import Popen, PIPE
+
+# external tools we use that come with visual studio sdk
```

Comment:
cmake needs to be called from a visual studio console window so the path is setup correctly to find dumpbin and undname.